.github/workflows/android.yml

```diff
@@ -1,16 +1,46 @@
 name: Android CI
-on: [pull_request]
 
-jobs:
-  build:
+on:
+  push:
+    branches:
+      - main
```

Comment:
I don't think we need a CI run when `main` is updated. All changes are made via pull requests. Lately I also rebase them onto `main`. But even without that, it's very unlikely there's a problem where tests pass on the PR branch, but fail when the branch is merged into main. At least I can't remember this happening in the past. If it does happen, the next pull request will catch that case.

Reply:
Reverted.

Reply:
I did like the `paths-ignore` lines.

Reply:
Done.